src/main/java/com/google/sps/data/QueryDatastore.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.data;
+
+import java.util.List;
+import java.util.ArrayList;
+import com.google.sps.data.Comment;
+import com.google.sps.data.Constants;
+import com.google.sps.data.Post;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Key; 
+import com.google.appengine.api.datastore.KeyFactory;
+
+public class QueryDatastore {
+    public static Entity queryForRepresentative(String repName) {
```

Comment:
why does this not throw an EntityNotFoundException? we should standardize across queryforRep and queryForPost

Reply:
nevermind this is not for ids - manually searching for name